recOrder/plugin/widget/main_widget.py

```diff
@@ -1551,19 +1552,33 @@ def connect_to_mm(self):
                 config_list = []
                 for j in range(configs.size()):
                     config_list.append(configs.get(j))
-                if np.all([ch in config_list for ch in calib_channels]):
-                    config_group_found = True
-                    self.config_group = group
+                if np.all([np.any([ch in config for config in config_list]) for ch in self.calib_channels]):
```

Comment:
Would you mind adding a comment here on the intended behaviour? This is a tricky section.

I think you're trying to catch cases where there's a typo in one of the 'stateX' strings when you're configuring micromanager?

Reply:
Just added some documentation above this section: 

https://github.com/mehta-lab/recOrder/blob/081e59ae5f6fa2077505b9980074c8b6a2530e2c/recOrder/plugin/widget/main_widget.py#L1583-L1589